tos.html

```diff
@@ -8,17 +8,17 @@ <h2 id="1-who-can-use-the-service">1. Who can use the Service</h2>
 </ul>
 <h2 id="2-member-conduct">2. Member Conduct</h2>
 <p>You agree not to use the Service to:</p>
-<ul>
+<ol>
     <li>attempt to gain unauthorized access to the Service or to any servers, systems, networks, databases, or other technology or data involved in its provision;</li>
     <li>violate any applicable law or regulation;</li>
     <li>post content which you do not have the right or permission to post, or to infringe any copyright or other proprietary right of any entity or person;</li>
     <li>post content that is hateful, discriminatory, harmful, illegal, or otherwise objectionable;</li>
     <li>impersonate any other person, group, or entity;</li>
     <li>violate the then-current Code of Conduct of the Service, which is available <a href="https://writing.codidact.com/help/code-of-conduct">in our help center</a>.</li>
-</ul>
+</ol>
 <h2 id="3-user-generated-content">3. User-Generated Content</h2>
 <p>If you sign up for an account, you will be able to submit questions, answers, comments, and other types of posts, some of which may include images, video, or other media (collectively, &quot;User-Generated Content&quot; or &quot;Content&quot;).</p>
-<p>By posting here, you license any such Content under the terms of one of (in increasing order of precedence) (a) the site default license at the time of creating your post; (b) the license of the category in which you post, or (c) the license you choose from the available list at the time of creating your post &mdash; the &quot;Applicable License&quot;. You represent and warrant that you own the rights to all Content you post or that you have the right or have been given permission from the relevant copyright holder to post it under the terms of the Applicable License.</p>
+<p>By posting here, you license any such Content under the terms of one of (in increasing order of precedence) (a) the site default license at the time of creating your post; (b) the license of the category in which you post, or (c) the license you choose from the available list at the time of creating your post &mdash; the &quot;Applicable License&quot;. You represent and warrant that you own the rights to all Content you post or that you have the right or have been given permission from the relevant copyright holder to post it under the terms of the Applicable License. You agree that any term of the Applicable License which requires attribution of such Content to you (an "Attribution Clause") shall be satisfied by the presentation of your username alongside the Content. You acknowledge that (a) the username presented as such will be your current username, not necessarily the username you used at the time of posting the Content, and (b) that your username may be changed by Us at any time if We in our sole discretion determine it to be unacceptable in line with clause 2.4 above, and you agree that these will not cause the cessation of satisfaction of any Attribution Clause.</p>
```

Comment:
LGTM.

Reply:
Oh, teeny tiny wording nitpick, only because this is legal language and just in case: "You acknowledge that (a)... , and (b) that ..." -- last "that" should go away (there's already a "that" in the outer clause).